### PR TITLE
Move kubevirt/kubevirt to new nodes

### PR DESCRIPTION
**Pull request by rmohr**
* migrate greenhouse and docker mirror to new nodes
 * trigger kubevirt/kubevirt jobs on new nodes only

/hold

### CODE REVIEW

**Comment by rmohr** [comment]
Because we don't want cross-zone cache traffic, the rollout order is like this:
 * temporary disable prow hook for kubevirt/kubevirt
 * stop all running jobs
 * move the cache by running ansible
 * merge this PR to update jobs
 * enable hook again

File: github/ci/prow/files/jobs/kubevirtci/kubevirtci-presubmits.yaml [hunk 1]
```diff
@@ -548,7 +548,7 @@ presubmits:
     spec:
       nodeSelector:
         model: r430
```
Comment (eedri) [inline github/ci/prow/files/jobs/kubevirtci/kubevirtci-presubmits.yaml]: should the model change as well? since its not r430 anymore

Comment (rmohr) [reply github/ci/prow/files/jobs/kubevirtci/kubevirtci-presubmits.yaml]: Yes, good catch. Thanks.

Comment (rmohr) [reply github/ci/prow/files/jobs/kubevirtci/kubevirtci-presubmits.yaml]: Fixed.

**Pull request merged**